app/models/frontend/expenses/tailoring/ExpensesTailoringCategoryTypeAnswer.scala

```diff
@@ -18,12 +18,12 @@ package models.frontend.expenses.tailoring
 
 import play.api.libs.json.{Json, OFormat, OWrites, Reads}
 
-final case class ExpensesTailoringNoExpensesAnswers(expensesCategories: ExpensesCategories)
+final case class ExpensesTailoringCategoryTypeAnswer(expensesCategories: ExpensesCategories)
 
-object ExpensesTailoringNoExpensesAnswers {
-  implicit val reads: Reads[ExpensesTailoringNoExpensesAnswers] = Json.reads[ExpensesTailoringNoExpensesAnswers]
+object ExpensesTailoringCategoryTypeAnswer {
```

Comment:
Do we need individually defined reads and writes? Looks like we can just use the formats (same applies to `ExpensesTailoringTotalAmountAnswers`)